components/transfer/Transfer.vue

```diff
@@ -379,46 +379,160 @@ const { fetchFiatPrice, getCurrentTokenValue } = useFiatStore()
 const { initTransactionLoader, isLoading, resolveStatus, status } =
   useTransactionStatus()
 const { toast } = useToast()
-const isTransferModalVisible = ref(false)
-const isLoaderModalVisible = ref(false)
+const { getTokenIconBySymbol } = useIcon()
+const { tokens } = useToken()
+const { chainExistentialDeposit } = useExistentialDeposit()
 
-watch(isLoading, (newValue, oldValue) => {
-  // trigger modal only when loading change from false => true
-  // we want to keep modal open when loading changes true => false
-  if (newValue && !oldValue) {
-    isLoaderModalVisible.value = isLoading.value
-  }
-})
+const DOT_BUFFER_FEE = 10000000 // 0.001
+const KSM_BUFFER_FEE = 100000000 // 0.0001
```

Comment:
**suggestion**: move to `libs/static`

Reply:
I'll do it in a follow up after this and #8740 are merged
don't want to create merge conflicts